pkg/apis/submariner.io/v1/types.go

```diff
@@ -65,6 +65,7 @@ type EndpointList struct {
 // +genclient
 // +genclient:noStatus
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:printcolumn:JSONPath=".status.haStatus",name="ha-status",description="High availability status of the Gateway",type="string"
```

Comment:
```suggestion
// +kubebuilder:printcolumn:JSONPath=".status.haStatus",name="HA Status",description="High availability status of the Gateway",type="string"
```

Reply:
I wondered about that; our current CRD uses `ha-status`, so I just copied that, but it would make sense to change it.

Reply:
yeah this becomes the user-visible column title.